test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -114,34 +136,5 @@ public void Dispose()
                 CallContextServiceLocator.Locator.ServiceProvider = _originalProvider;
             }
         }
-
-        // TODO: Kill this code
```

Comment:
You can resolve this https://github.com/aspnet/Mvc/issues/1612 as part of this PR